vstruct/primitives.py

```diff
@@ -30,15 +30,22 @@ def vsReverseMapping(self, val, default=None):
 
 
 class v_base(object):
-    def __init__(self):
+    def __init__(self, bigend=False):
         self._vs_meta = {}
+        self._vs_bigend = bigend
 
     def vsGetMeta(self, name, defval=None):
         return self._vs_meta.get(name, defval)
 
     def vsSetMeta(self, name, value):
         self._vs_meta[name] = value
 
+    def vsGetEndian(self):
```

Comment:
This is named a bit more generically than the implementation ( returns True if big endian ). Could probably just rename `vsSetBigEndian()` and `vsIsBigEndian()` to make them self evident.

Reply:
apparently missed a couple comments
we can make those changes.  we have this naming peppered throughout the codebase.  ENDIAN_MSB is 1 and ENDIAN_LSB is 0, so we use getEndian in `envi` and `envi.memory` and `vivisect`.  in order to make this change meaninful, we'd need to change all the APIs to match.

Reply:
Ah, gotcha. I'm ok with it as long as we've been externally using ENDIAN_MSB and ENDIAN_LSB. Maybe add a quick doc string so it's clear :+1: